mojo/edk/system/node_controller.cc

```diff
@@ -233,8 +234,9 @@ void NodeController::SendBrokerClientInvitation(
 
 void NodeController::AcceptBrokerClientInvitation(
     ConnectionParams connection_params) {
-#if !defined(CASTANETS)
-   DCHECK(!GetConfiguration().is_broker_process);
+#if defined(CASTANETS)
+   if (base::Castanets::IsEnabled())
```

Comment:
I think this line should be if (!base::Castanets::IsEnabled()).

Reply:
right, will update patchset